README.md

```diff
@@ -1,10 +1,22 @@
 ## Workshop 1 - Password Management
-__Learning Outcomes__
 
-- Why not to store plaintext passwords.
-- What is hashing?
-- Inneffective hashing vs bcrypt hashing.
-- How to use 'bcryptjs' npm package.
+### Learning outcomes
+Understand:
++ the importance of making passwords non-recoverable from a database
```

Comment:
i prefer 'why to avoid storing plaintext passwords'. 'non-recoverable from a database' is a bit alien